webanno-api-annotation/src/main/java/de/tudarmstadt/ukp/clarin/webanno/api/annotation/rendering/model/VDocument.java

```diff
@@ -137,4 +137,14 @@ public VObject get(VID aVid)
     {
         return Collections.unmodifiableCollection(annotationLayers.values());
     }
+
+    public void add(VObject aVobj)
+    {
+        if (aVobj instanceof VSpan) {
+            add((VSpan) aVobj);
+        }
+        else if (aVobj instanceof VArc) {
+            add((VArc) aVobj);
+        }
```

Comment:
This method should have an additional `else` block where an `IllegalArgumentException` is thrown when the object is neither a span nor an arc.